rules/windows/process_creation/proc_creation_win_defender_exclusions_via_powershell.yml

```diff
@@ -0,0 +1,39 @@
+title: Windows Defender Exclusions Added via Powershell
+id: 801f7258-40a8-4c10-bf4b-d76248556aac
+related:
+    - id: c1344fa2-323b-4d2e-9176-84b4d4821c88
+      type: similar
```

Comment:
We already cover this case of proc creation in 17769c90-230e-488b-a463-e05c08e9d48f